addons/mail/static/src/messaging/component/chatter_topbar/chatter_topbar.xml

```diff
@@ -40,8 +40,8 @@
                     </t>
                 </button>
                 <div class="o-autogrow"/>
-                <t t-if="chatter.hasFollowers">
-                    <div class="o_ChatterTopbar_rightSection">
+                <div class="o_ChatterTopbar_rightSection" t-att-class="{ 'o-has-flap': props.hasCloseButton }">
```

Comment:
Am I blind? I didn't see the `o-has-flap` being used anywhere.

Reply:
You're not, it's old code I forgot to remove